src/plugins/injection/README.md

```diff
@@ -0,0 +1,325 @@
+- infos = Information about the injection plugin is below
+- infos/author = Markus Raab <elektra@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = inject
+- infos/recommends =
+- infos/placements = presetstorage
+- infos/status = maintained nodep
+- infos/metadata = inject/randSeed inject/structure/sectionRemove inject/structure/sectionDuplicate inject/structure/sectionReallocate inject/semantic inject/resource inject/typo/transposition inject/typo/insertion inject/typo/caseToggle inject/typo/deletion inject/typo/changechar inject/typo/space inject/domain inject/limit/min inject/limit/max
```

Comment:
please avoid camel case, we always use / for separation. So it would be inject/rand/seed, ...